.github/workflows/tester.yml

```diff
@@ -0,0 +1,34 @@
+name: "Tester"
+
+on: [pull_request]
+jobs:
+  tests:
+    name: Tester
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 2
+
+    - run: git checkout HEAD^2
```

Comment:
similar comment as above

Reply:
Not sure, copied it from somewhere. Replacing with what we have in release.yml, without this twos.